Service/Runner.php

```diff
@@ -320,6 +343,7 @@ private function runImages($jobId, $configId, Component $component)
             $output = $container->run();
             if ($image->isMain()) {
                 $componentOutput = $output->getOutput();
+                $this->usageFile->storeUsage();
```

Comment:
@odinuv tu su myslel try-finally na ten output? .... lebo na storeUsage to asi moc vyznam nema, pretoze, ked to skonci chybou, tak ho proste nezapiseme do ES